src/main/kotlin/sk/streetofcode/webapi/service/CourseServiceImpl.kt

```diff
@@ -137,6 +139,7 @@ class CourseServiceImpl(
                     getProgressMetadata(it.id)
                 )
             }
+            .sortedBy { it.courseOrder }

```

Comment:
I think that sorting courses in this level is good, because FE doesn't even need to know about course orders. I don't see no harm to sort in these level, do you? Would you rather sort in FE? @gabrielKerekes

Reply:
The thought of sorting on FE came to my mind while reviewing the FE PR, but IMO it doesn't matter for our usecase at this point. If we ever need some better ordering/filtering etc. than it might be worth a discussion.

Reply:
On another note - I think we could and should reuse `getAllCoursesOverview` and just do the `.filter` on its result here.

Reply:
That's a good point